spec/client_spec.js

```diff
@@ -574,6 +575,19 @@ describe('Client', function() {
         clock.restore();
         expect(promise).to.be.rejectedWith(Error, 'Invocation request timeout').and.notify(done);
       });
+
+      it('doesnt reject whitelisted promises after 5 seconds', function(done) {
+        var clock = sinon.useFakeTimers();
+        promise = subject.invoke('instances.create');
+        clock.tick(10000);
+        clock.restore();
+        expect(promise).to.eventually.become({ errors: {}, 'ticket.appendText': true }).and.notify(done);
```

Comment:
`ticket.appendText` is confusing here, can we use the name of the request 'instances.create'?